src/tests/functional/plugin/shared/include/behavior/compiled_model/import_export.hpp

```diff
@@ -518,6 +518,75 @@ TEST_P(OVClassCompiledModelImportExportTestP, smoke_ImportNetworkNoThrowWithDevi
     OV_ASSERT_NO_THROW(executableNetwork.create_infer_request());
 }
 
+//
+// GetRuntimeModel
+//
+typedef std::tuple<ov::element::Type,  // Element type
+                   ov::Shape,          // Shape
+                   std::string         // Device name
+                   >
+    OVCompiledModelGraphUniqueNodeNamesTestParams;
+
+class OVCompiledModelGraphUniqueNodeNamesTest
```

Comment:
Should it be instantiated in API conformance? Please, add test instance in `ov_api_conformance_tests`

Reply:
Is there any policy to guide which test cases should be instantiated in API conformance test, which test cases should be in cpu/gpu func tests?

Reply:
The example of instantiation for `ov|_plugin` -> https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/src/ov_plugin/life_time.cpp

These tests are out of mandarory scope. Please place `ov_plugin` instead of `ov_plugin_mandatory` as a test instance prefix

Reply:
Do you mean we need change life_time.cpp test instantiates to ov_plugin?